aws-rds/README.md

```diff
@@ -24,81 +24,181 @@ Use SignalFx to monitor Amazon Relational Database Service (RDS) via [Amazon Web
+blank.
+* Make sure the `Enabled` switch is activated.
+
+Click `Add`, then click `Save` in the upper right corner.
+
+That's it! Your metrics are on the way to SignalFx ingest!
+
+## Building from source
+
+### 1. Set up the execution role
+The execution role just needs basic Lambda execution permissions and KMS
+decrypt permissions (if you wish to encrypt your SignalFx access token). If you
+don't want to create one, you can select from a list of templates when you
+create the lambda function.
+
+### 2. Set up an encryption key and encrypt access token
+Only follow this step if you chose to encrypt your access token. Either create
+a new KMS encryption key or select a preexisting one. **The key must be in the
+same availability zone as the RDS instances you are monitoring.** You can
+create and manage encryption keys from IAM in the AWS management console.
+Documentation on KMS encryption from the CLI can be found
+[here](http://docs.aws.amazon.com/cli/latest/reference/kms/encrypt.html).
+Make sure you have access to the cipher text output by the encryption as well
+as the key id of the encryption key you used.
+
+### 3. Clone the source repo and build the deployment package
+You can find the repo
+[here](https://github.com/signalfx/enhanced-rds-monitoring).
+After you have cloned the repo, do the following:
+```
+$ cd enhanced-rds-monitoring
+$ ./build.sh
+```
+The package will be named `enhanced_rds.zip`. This is the file to upload for the Lambda.
+
+### 4. Create and configure the Lambda function
+From the Lambda creation screen, make sure you have selected
+`Build from scratch`. Select a name for your function. For `Runtime` select
+`Python3.8` (although `Python3.6` and `Python3.7` are also supported). For the execution role, either select the role you want to use or
+select `Create from Template` and add KMS decrypt permissions if necessary. You
+will also need to choose a name for the role.
+
+For subsequent tabs, follow the instructions below.
+
+#### Designer
+The only thing to be done here is to set up the trigger from CloudWatch Logs:
+1. Select CloudWatch Logs from the list on the left. A section labelled
+`Configure triggers` is displayed. For the `Log group` field, select
+`RDSOSMetrics`.
+2. Choose a filter name, but leave the filter
+pattern blank. You can disable the trigger to start if you want (though you
+will need to manually enable it later to start sending metrics)
+3. Click Add.
+
+#### Function code
+Once the function is created you can change the configurations: Upload the ZIP
+file containing the deployment package, then change the text in `Handler` to be
+`enhanced_rds.lambda_script.lambda_handler`.
+
+#### Environment variables
+
+First create an environment variable called `groups`. This will store the list of metric groups to be reported. To report all available metrics, enter `All`. Otherwise, list the names of desired metric groups, spelled exactly as above, separated by single spaces.
```

Comment:
I would format this section as a list. These are tasks and are probably easier for users to read in a list format.